oauth2_http/javatests/com/google/auth/oauth2/MockHttpTransportFactory.java

```diff
@@ -0,0 +1,15 @@
+package com.google.auth.oauth2;
+
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.testing.http.MockHttpTransport;
+import com.google.auth.http.HttpTransportFactory;
+
+class MockHttpTransportFactory implements HttpTransportFactory {
```

Comment:
If you plan to use it in other repos, this needs to be a public class.

Reply:
Seems like it was the only exposed class that needed a modifier. Changed